src/mrinufft/operators/base.py

```diff
@@ -485,9 +484,9 @@ def __repr__(self):
         )
 
     @classmethod
-    def with_autograd(cls, variable, *args, **kwargs):
+    def with_autograd(cls, wrt_data, wrt_traj, *args, **kwargs):
```

Comment:
Add a docstring, and default values for the arguments. 
```suggestion
    def with_autograd(cls, *args, wrt_data=True, wrt_traj=False, **kwargs):
```